internal/frontend/http/ui.go

```diff
@@ -362,18 +362,26 @@ func (p WizardParams) URLValues() url.Values {
 
 // wizardVersions handles the 'pick Talos version' step.
 func (f *Frontend) wizardVersions(ctx context.Context, params WizardParams) (string, any, url.Values, error) {
-	versions, err := f.getTalosVersions(ctx, params.SelectedVersion)
+	versions, err := f.getTalosVersions(ctx, params.SelectedVersion, params.Target)
 	if err != nil {
 		return "", nil, nil, err
 	}
 
+	var message string
+
+	if params.Target == TargetSBC {
+		message = "Only Talos 1.7+ is supported with SBC's"
```

Comment:
I think this should be pushed down to the template and translated (we can make same condition in the template)